index.html

```diff
@@ -80,3 +76,221 @@ <h2 class="secondary-header">LET’S CONNECT</h2>
+  <!-- first project  -->
+ <div class="projects container">
+    <img class="project-img" src="images/Snapshoot Portfolio (1).png" alt="" />
+    <h3 class="tertiary-heading">Tonic</h3>
+    <ul class="project-details">
+      <li>CONOPY</li>
+      <li>Back End Dev</li>
+      <li>2015</li>
+    </ul>
+    <p class="project-description">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required.
+    </p>
+    <ul class="project-lang">
+      <li>HTML</li>
+      <li>CSS</li>
+      <li>JavaScript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+  <!-- second project  -->
+  <div class="projects container">
+    <img class="project-img" src="images/Snapshoot Portfolio.png" alt="" />
+    <h3 class="tertiary-heading">Multi-Post Stories</h3>
+    <ul class="project-details">
+      <li>CONOPY</li>
+      <li>Back End Dev</li>
+      <li>2015</li>
+    </ul>
+    <p class="project-description">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required.
+    </p>
+    <ul class="project-lang">
+      <li>HTML</li>
+      <li>CSS</li>
+      <li>JavaScript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+  <!-- thirs project  -->
+  <div class="projects container">
+    <img class="project-img" src="images/Snapshoot Portfolio (1).png" alt="" />
+    <h3 class="tertiary-heading">Tonic</h3>
+    <ul class="project-details">
+      <li>CONOPY</li>
+      <li>Back End Dev</li>
+      <li>2015</li>
+    </ul>
+    <p class="project-description">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required.
+    </p>
+    <ul class="project-lang">
+      <li>HTML</li>
+      <li>CSS</li>
+      <li>JavaScript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+  <!-- fourth project  -->
+  <div class="projects container">
+    <img class="project-img" src="images/Snapshoot Portfolio.png" alt="" />
+    <h3 class="tertiary-heading">Multi-Post Stories</h3>
+    <ul class="project-details">
+      <li>CONOPY</li>
+      <li>Back End Dev</li>
+      <li>2015</li>
+    </ul>
+    <p class="project-description">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required.
+    </p>
+    <ul class="project-lang">
+      <li>HTML</li>
+      <li>CSS</li>
+      <li>JavaScript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+</section>
+<!-- ABOUT ME SECTION  -->
+<section class="about container">
+  <h2 class="secondary-header">About Myself</h2>
+  <p class="headline-description">
+    Hello I’m a software developer! I can help you build a product , feature
+    or website. Look through some of my work and experience! If you like
+    what you see and have a project you need coded, don’t hesitate to
+    contact me.
+  </p>
+  <div class="social-media">
+    <h2 class="social-header">LET’S CONNECT</h2>
```

Comment:
- [ ] [OPTIONAL] I would highly recommend that you change the color of "LET’S CONNECT" title from black to `#7F8CFF`  and use light font weight, as we are aiming for a pixel perfect design. Also please add a bit more `margin-top` and `margin-bottom` to match the layout of the [figma design.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=W0MZ6qGGoBjUhLi8-0)

|. Figma design |. You design |
|---------------|-------------|
|<img width="243" alt="Screenshot 2023-04-30 at 3 03 39 AM" src="https://user-images.githubusercontent.com/44798044/235325946-299e053c-e44d-4f96-b44a-10b69c3ef77f.png">|<img width="226" alt="Screenshot 2023-04-30 at 3 02 31 AM" src="https://user-images.githubusercontent.com/44798044/235325953-b53d651a-fe49-4afe-b02b-f5591a75ee62.png">|